src/Support/helpers.php

```diff
@@ -198,7 +198,22 @@ function plugins_path($path = '')
      */
     function uploads_path($path = '')
     {
-        return app('path.uploads').($path ? '/'.$path : $path);
+        $uploads_path = rtrim(Config::get('cms.storage.uploads.path', app('path.uploads')), DIRECTORY_SEPARATOR);
+        return $uploads_path.($path ? DIRECTORY_SEPARATOR.ltrim($path, DIRECTORY_SEPARATOR) : $path);
```

Comment:
```suggestion
        return $uploads_path . ($path ? DIRECTORY_SEPARATOR . ltrim($path, DIRECTORY_SEPARATOR) : $path);
```

Reply:
@mjauvin Do you need to do this path trimming on the other helpers too?

Reply:
@bennothommo Yes, all gonna be handled by the next commit...